Library/Formula/ledger.rb

```diff
@@ -96,22 +65,7 @@ def install
     assert_equal 0, $?.exitstatus
 
     if build.with? "python"
-      system "python", "#{share}/ledger/demo.py"
+      system "python", share/"ledger/demo.py"
```

Comment:
Thanks @bfontaine, I've updated the pull request accordingly with consistent use of `pkgshare` over `share+"ledger"`.